oximeter/db/src/client/mod.rs

```diff
@@ -4185,4 +4185,242 @@ mod tests {
         db.cleanup().await.unwrap();
         logctx.cleanup_successful();
     }
+
+    // The schema directory, used in tests. The actual updater uses the
+    // zone-image files copied in during construction.
+    const SCHEMA_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/schema");
```

Comment:
These tests seem more integration-like, but we don't separate things into a tests directory within the crate, right?

Reply:
Not right now, no. I don't really have a strong opinion for these ones.

Reply:
@bnaecker yep. It's def fine for now.

Reply:
Oh, actually these might need to be in the crate. They use the private `Client::execute_with_body()` method for running the raw SQL that fetches the database tables.